module_3/gift_certificates_system/model/src/main/java/com/epam/esm/listener/GiftCertificateListener.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.listener;
+
+import com.epam.esm.entity.GiftCertificate;
+
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import java.time.LocalDateTime;
+
+
+public final class GiftCertificateListener {
+    private static LocalDateTime currentDateTime;
```

Comment:
Why did you define this as class field?

Reply:
Because `currentDateTime` used in `beforeCreate()` and `beforeUpdate()` methods.